src/lib_c/x86_64-windows-msvc/c/sys/types.cr

```diff
@@ -2,4 +2,9 @@ lib LibC
   alias InoT = UShort
   alias DevT = UInt
   alias OffT = Long
+
+  alias Word = UInt16
+  alias WChar = UInt16
+  alias DWord = UInt32
+  alias BOOL = Int32
```

Comment:
These are all in `winapi.cr` (which I actually think should be removed, would love to head the core team's opinion on this)

Reply:
That depends on where they're expected to be defined in windows includes. AFAIK there is no `winapi.h`.

Reply:
`DWORD`is declared in `IntSafe.h`, `WCHAR` in `WinNT.h`, `WORD` and `BOOL` in `WinDef.h`

https://msdn.microsoft.com/en-us/library/windows/desktop/aa383751(v=vs.85).aspx

Reply:
Well I don't think they should be in `sys/types`, and if you *are* going to define these types in the PR, they should at least be removed from `winapi.cr`

Reply:
I'll put them in their respective locations.